hooks/lib/gcr-registry-provider.bash

```diff
@@ -0,0 +1,25 @@
+login() {
+  # Currently assume the use of docker-credential-gcr to manage AuthN transparently.
```

Comment:
good idea. We could probably do this for ECR too, however it would mean consumers have to "do more" to use the plugin

Reply:
Yes, definitely; I'll flag that in the readme but also not address it immediately. I'm just getting to the point internally to actually _try_ this for the first time, it has been a hefty week.

Reply:
👍 we can certainly tackle in a new PR.